plugins/org.jkiss.dbeaver.data.gis.view/OSGI-INF/l10n/bundle_de.properties

```diff
@@ -1,10 +1,13 @@
 Bundle-Vendor = DBeaver Corp
-Bundle-Name = DBeaver Data GIS - ansicht
+Bundle-Name = DBeaver Data GIS Ansicht
 
 presentation.geometry.name = Karte
 presentation.geometry.description = Geodaten: Geometrie- oder Geografieobjekte
 
 extension-point.org.jkiss.dbeaver.ui.gis.panel.GISTextViewer.label = Text
 extension-point.org.jkiss.dbeaver.ui.gis.panel.GISTextViewer.description = Zeigen Sie das GIS-Objekt in Textform an
 extension-point.org.jkiss.dbeaver.ui.gis.panel.GISBrowserViewer.label = Karte
-extension-point.org.jkiss.dbeaver.ui.gis.panel.GISBrowserViewer.description = GIS-Objekt auf der Karte anzeigen
\ No newline at end of file
+extension-point.org.jkiss.dbeaver.ui.gis.panel.GISBrowserViewer.description = GIS-Objekt auf der Karte anzeigen
+
+pref.page.name.gis = GIS Betrachter
+keyword.org.jkiss.dbeaver.preferences.gis = GIS SRID CRS Spatial
```

Comment:
Could you please also add a translate of the word "Spatial" here?

Reply:
Spatial is a special word that is also in germany widely used to describe geo data. But I tried a translation